code/modules/projectiles/ammo_datums.dm

```diff
@@ -2564,24 +2564,28 @@ datum/ammo/bullet/revolver/tp44
 	hud_state = "flame"
 	hud_state_empty = "flame_empty"
 	damage_type = BURN
-	flags_ammo_behavior = AMMO_INCENDIARY|AMMO_IGNORE_ARMOR|AMMO_FLAME
+	flags_ammo_behavior = AMMO_INCENDIARY|AMMO_IGNORE_ARMOR|AMMO_FLAME|AMMO_EXPLOSIVE
 	armor_type = "fire"
-	max_range = 6
-	damage = 50
+	max_range = 7
+	damage = 0
 	bullet_color = LIGHT_COLOR_FIRE
 	var/fire_color = "red"
 	var/burnlevel = 31
 	var/burntime = 17
-	var/fire_delay = 20
+
+/datum/ammo/flamethrower/drop_flame(turf/T)
+	if(!istype(T))
```

Comment:
What is that ? Why is it needed

Reply:
This is actually just me cargo coding from the tank flamer code. I think it's for firing past the edge of the map.